tests/run-tests.ps1

```diff
@@ -47,5 +47,30 @@ else {
     $env:LOCAL_RUN = $null
 }
 
-& dotnet test --filter Category=$TestCategory -c Release --logger:trx $PSScriptRoot/Microsoft.DotNet.Framework.Docker.Tests/Microsoft.DotNet.Framework.Docker.Tests.csproj
+# Powershell treats each variable passed to the command line call as an individual argument.  But we're constructing the $testFilter variable
```

Comment:
It feels like this logic would be simpler in a way to use the [Exec function](https://github.com/dotnet/docker-tools/blob/master/eng/common/Invoke-ImageBuilder.ps1#L45) we have in the common scripts.   The logic would build up a single Args string.  An added bonus is the Exec method would emit out the command that is being executed which would be helpful diagnostic info.